sciLifeLab_utils/run_assemblies.py

```diff
@@ -89,54 +89,20 @@ def main(args):
             sample_files.remove(pair2_file)
             library += 1
         sample_YAML.close
-        if not hasattr(args, "email"):
-            args.email = None
-        submit_job(sample_YAML_name, args.global_config, sample_dir_name ,
-                pipeline, args.env, args.email, args.time, args.project,
-                args.threads, args.qos) # now I can submit the job to slurm
-        os.chdir(projectFolder)
 
+        #Run the job
+        all_modules = {
```

Comment:
Hmm. So there we might not need to load the modules you're saying? I haven't experimented with it, just trying to replicate what we already do.

What do you suggest?